pkg/resources/share.go

```diff
@@ -146,7 +146,8 @@ func ReadShare(data *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
-	accs := strings.Split(to.String, ", ")
+	// accs := strings.Split(to.String, ", ")
```

Comment:
should this be rmd?

Reply:
hmm, on second thought, why not keep strings.Split but change ", " to ","?

Reply:
yeah will do later